meta-avocado/recipes-avocado/packagegroups/packagegroup-avocado-extra.bb

```diff
@@ -2,7 +2,9 @@ DESCRIPTION = "Packagegroup for Avocado extra"
 LICENSE = "Apache-2.0"
 
 inherit packagegroup
+PACKAGE_ARCH = "${MACHINE_ARCH}"
```

Comment:
I wonder what implications this has on all the packages being built.

Does this mean Yocto will compile with all the tunes for a target `MACHINE_ARCH` whenever possible?

Reply:
Unclear. This was added in response to the dreaded, "allarch dynamic renamed package error". I am open to other ways of solving this.